cmd/utils/flags.go

```diff
@@ -949,8 +960,26 @@ func setIPC(ctx *cli.Context, cfg *node.Config) {
 	}
 }
 
-// setUltraLight configures the ultra light client settings from the command line flags.
-func setUltraLight(ctx *cli.Context, cfg *eth.Config) {
+// setLes configures the les server and ultra light client settings from the command line flags.
+func setLes(ctx *cli.Context, cfg *eth.Config) {
+	if ctx.GlobalIsSet(LightLegacyServFlag.Name) { // Deprecated in favor of light.maxpeers, remove in 2021
```

Comment:
`// Deprecated in favor of light.serve, remove in 2021`

Reply:
Good catch, removed altogether.